ebms-send-in/src/test/kotlin/no/nav/emottak/fellesformat/FellesFormatWrapperTest.kt

```diff
@@ -13,23 +18,52 @@ class FellesFormatWrapperTest {
     fun wrapMessageInPasientlisteForesporselEIFellesFormat() {
         val sendInRequest = validSendInPasientlisteRequest.value
         val fellesFormat = wrapMessageInEIFellesFormat(sendInRequest)
+        val refParam = refParam(fellesFormat)
         Assertions.assertEquals(fellesFormat.mottakenhetBlokk.partnerReferanse, sendInRequest.cpaId)
+        log.info(sendInRequest.marker(), "refParam ${birthDay(refParam)}")
+        Assertions.assertEquals(fellesFormat.mottakenhetBlokk.partnerReferanse, sendInRequest.cpaId)
+        log.info(marshal(fellesFormat))
+    }
+
+    @Test
+    fun wrapMessageInHarBorgerEgenandelFritakREIFellesFormat() {
+        68
+        val sendInRequest = validSendInHarBorgerEgenandelfritakRequest.value
+        val fellesFormat = wrapMessageInEIFellesFormat(sendInRequest)
+        val refParam = refParam(fellesFormat)
+        Assertions.assertEquals(fellesFormat.mottakenhetBlokk.partnerReferanse, sendInRequest.cpaId)
+        log.info(sendInRequest.marker(), "refParam ${birthDay(refParam)}")
         log.info(marshal(fellesFormat))
     }
 
     @Test
     fun wrapMessageInHarBorgerFrikortREIFellesFormat() {
         val sendInRequest = validSendInHarBorgerFrikortRequest.value
         val fellesFormat = wrapMessageInEIFellesFormat(sendInRequest)
+        val refParam = refParam(fellesFormat)
+        Assertions.assertEquals(fellesFormat.mottakenhetBlokk.partnerReferanse, sendInRequest.cpaId)
+        log.info(sendInRequest.marker(), "refParam ${birthDay(refParam)}")
+        log.info(marshal(fellesFormat))
+    }
+
+    @Test
+    fun wrapMessageInHarBorgerFrikortMengdeREIFellesFormat() {
+        val sendInRequest = validSendInHarBorgerFrikortMengdeRequest.value
+        val fellesFormat = wrapMessageInEIFellesFormat(sendInRequest)
+        val refParam = refParam(fellesFormat)
         Assertions.assertEquals(fellesFormat.mottakenhetBlokk.partnerReferanse, sendInRequest.cpaId)
+        log.info(sendInRequest.marker(), "refParam $refParam")
         log.info(marshal(fellesFormat))
     }
 
     @Test
     fun wrapMessageInInntektforesporselEIFellesFormat() {
+        // val sendInRequest = validSendInInntektforesporselRequestWithENH.value
```

Comment:
Hvis denne kodebiten ikke lenger er nødvendig og PR-en er klar for review, kan du gjerne fjerne den. Hvis PR-en fortsatt er under arbeid, kan det være lurt å sette den som en 'Draft' PR. :thought_balloon: